src/components/common/ToggleButton/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+import cn from 'utils/ts/classNames';
+import styles from './ToggleButton.module.scss';
+
+interface ToggleButtonProps {
+  className: string;
+  firstState: boolean;
+  toggleFn: () => void;
+}
+
+export default function ToggleButton({
+  className, firstState, toggleFn,
+}: ToggleButtonProps): JSX.Element {
+  const [toggle, setToggle] = useState(firstState);
+
+  const handleToggle = () => {
+    setToggle((prevState) => !prevState);
+    toggleFn();
+  };
```

Comment:
toggle의 초기값을 지정하지 않아도 false를 default 값으로 가져갈수있도록 조정하는건 어떤가요? 
ex)
```suggestion
  const [toggle, setToggle] = useState<boolean>(!!firstState || false);
```


사실 저 부분은 `useBooleanState` hook 을 사용한다면, 단번에 해결될거같긴해요ㅎㅎ
(심지어 toggle 이라는 상태변경 함수도 있네요)
```suggestion
  const {value: toggle, toggle: changeToggle} = useBooelanState(firstState);
```

Reply:
왜 바깥에는 useBooleanState 사용했으면서 정작 이 컴포넌트에는 안 썼을까요? 하하
수정하고 변수명도 조금씩 바꿨습니다.
651e27ef7245a479606c4fed71d3a5f86c10b13b